### ISSUE TITLE
Dataset not available

### ISSUE THREAD

**Issue by aheuser** [rodrigosantis1/backorder_prediction#1]
It would be great to try and reproduce your results, but the link to the dataset is dead. Would it be possible to post the data? Thank you.

**Comment by lrconcepcion** [comment]
Yes, the dataset link is dead. We'll be most grateful if you could post the data. Thanks so much!

**Comment by lrconcepcion** [comment]
> 
> 
> It would be great to try and reproduce your results, but the link to the dataset is dead. Would it be possible to post the data? Thank you.

Found the link: https://github.com/h2oai/h2o-tutorials/raw/master/h2o-world-2017/automl/data/product_backorders.csv 

See the original  Pandey (2019) article https://towardsdatascience.com/a-deep-dive-into-h2os-automl-4b1fe51d3f3e
